database.go

```diff
@@ -1,37 +1,81 @@
 package main
 
 import (
-	"database/sql"
+	"errors"
+	"fmt"
+	"strings"
 )
 
-var db *sql.DB
+var (
+	ErrDBConnStrParseNoSep = errors.New("Missing ':' in connection string")
+	ErrDBConnStrParseNotImplemented = func(dbName string) error {
+		return errors.New(fmt.Sprintf("The database '%s' is not implemented", dbName))
+	}
+)
 
-func LoadDatabase(dbFile string) error {
-	var err error
-	db, err = sql.Open("sqlite3", dbFile)
+type Database interface {
+	CommentService
+}
+
+func parseConnectionStr(connectionStr string) (dbName string, dbParams map[string]interface{}, err error) {
```

Comment:
Could you document an example `connectionStr` input as a comment? It'd help reviewing and might help future devs too.